s3/replication/replicator/src/s3replicator/app.py

```diff
@@ -17,11 +17,106 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-from s3replicationcommon.log import Logger
+# import sys
+# import argparse
+# import json
+from aiohttp import web
+import logging
+from .config import Config
+# from urllib.parse import urlparse, parse_qs
+from s3replicationcommon.log import setup_logging
+
+LOG = logging.getLogger('replicator_proc')
+
+HOST = '127.0.0.1'
+PORT = 8080
+
+# Dictionary holding job_id and fdmi record
+# e.g. : jobs = {"job1": {"obj_name": "foo"}}
+jobs = {}
+
+# List of active subscriber
+schedulers = []
+
+# List of jobs in progress
+jobs_inprogress = []
+
+# Route table declaration
+routes = web.RouteTableDef()
+
+@routes.get('/jobs')
+async def list_jobs(request):
+    """List_jobs
+
+    Handler to list in-progress jobs
+
+    """
+    LOG.debug(jobs_inprogress)
```

Comment:
Print only count to log. Also on side note please see all the comments in manager/app.py as most of those apply here.

Reply:
done